jboss/container/wildfly/launch/mp-config/added/launch/mp-config.sh

```diff
@@ -34,7 +25,7 @@ configure_microprofile_config_source() {
           quit
       end-if
 
-      /subsystem=microprofile-config-smallrye/config-source=config-map:add(dir={path=${MICROPROFILE_CONFIG_DIR}}, ordinal=${MICROPROFILE_CONFIG_DIR_ORDINAL})
+      /subsystem=microprofile-config-smallrye/config-source=config-map:add(dir={path="${MICROPROFILE_CONFIG_DIR}"}, ordinal=${MICROPROFILE_CONFIG_DIR_ORDINAL:-500})
```

Comment:
This is not really an issue so I haven't opened a WFWIP issue for this. Image.yaml file of the openshift image is already initializating this value, however it should be done here if that initialization is removed from the image.yaml, which makes sense to me. Microprofile script must be configured here, not in the image.yaml file.